spinta/manifests/tabular/helpers.py

```diff
@@ -859,6 +859,46 @@ def leave(self) -> None:
         pass
 
 
+class UniqueReader(TabularReader):
+    type: str = 'unique'
+
+    def read(self, row: ManifestRow) -> None:
+        reader = self.state.stack[-1]
+
+        if not isinstance(reader, (
+            DatasetReader,
+            ResourceReader,
+            BaseReader,
+            ModelReader,
+            PropertyReader,
+        )):
+            return
```

Comment:
If `unique` is added not on a `model`, then we should emit an error before return.